stories/topics/proc-aap-gcp-backup-and-restore.adoc

```diff
@@ -0,0 +1,137 @@
+[id="proc-aap-gcp-backup-and-restore"]
+
+= Ansible Automation Platform Backup and Restore Process
+
+To backup and restore your AAP deployment, it is vital to have your existing AAP Admin Secret name and value recorded somewhere safe. In addition, it is also extremely important that regular manual backups of the Cloud SQL DB instance and filestore backups are taken to ensure a deployment can be restored as close as possible to its previous working state. The playbooks backup and restore provides backup and restore support for the AAP on GCP foundation deployment. 
```

Comment:
To backup and restore your {PlatformNameShort} deployment,

your existing {PlatformNameShort} administration secret name

support for the {AAPonGCP} foundation deployment.